debugger/src/main/kotlin/org/rust/debugger/settings/RsDebuggerDataViewConfigurableUi.kt

```diff
@@ -24,17 +26,26 @@ class RsDebuggerDataViewConfigurableUi : ConfigurableUi<RsDebuggerSettings> {
         GDBRenderers.values().forEach { addItem(it) }
     }
 
+    private val decorateMsvcTypeNames: JBCheckBox = JBCheckBox(
+        RsDebuggerBundle.message("settings.rust.debugger.decorate.msvc.type.names.checkbox"),
+        RsDebuggerSettings.getInstance().decorateMsvcTypeNames
+    )
+
     override fun isModified(settings: RsDebuggerSettings): Boolean =
-        lldbRenderers.selectedIndex != settings.lldbRenderers.ordinal || gdbRenderers.selectedIndex != settings.gdbRenderers.ordinal
+        lldbRenderers.selectedIndex != settings.lldbRenderers.ordinal
+            || gdbRenderers.selectedIndex != settings.gdbRenderers.ordinal
+            || settings.decorateMsvcTypeNames != decorateMsvcTypeNames.isSelected
```

Comment:
```suggestion
            || decorateMsvcTypeNames.isSelected != settings.decorateMsvcTypeNames
```

nit: just to be consistent